echion/threads.h

```diff
@@ -172,6 +174,10 @@ void ThreadInfo::unwind(PyThreadState* tstate)
                 // We failed to unwind tasks
             }
         }
+        if (greenlet_thread_map.find(native_id) != greenlet_thread_map.end())
+        {
+            unwind_greenlets(tstate);
+        }
```

Comment:
So here, we always unwind python stack, then asyncio task stacks and then greenlet stacks. 

What would happen if there's a program that calls async function from a greenlet?

```Python
import asyncio
from greenlet import greenlet

async def main_async():
    print("Inside async function")
    await asyncio.sleep(1)
    print("Exiting async function")

def run_in_greenlet():
    print("Inside run_in_greenlet")
    loop = asyncio.new_event_loop()
    asyncio.set_event_loop(loop)
    loop.run_until_complete(main_async())
    loop.close()
    print("Exiting run_in_greenlet")

g = greenlet(run_in_greenlet)
g.switch()

```

Reply:
Good point. Here there is the tacit assumption that the two async frameworks are not mixed together (the current implementation in ddtrace gives precedence to asyncio in this case). I think it's best to keep things simple here for now, unless there is a substantial demand for supporting this scenario.

Reply:
Cool, mind adding a comment then?